schemas/org.mate.interface.gschema.xml

```diff
@@ -75,6 +75,11 @@
       <summary>List of symbolic names and color equivalents</summary>
       <description>A '\n' separated list of "name:color" as defined by the 'gtk-color-scheme' setting</description>
     </key>
+    <key name="gtk-color-palette" type="s">
+      <default>'#FF0000:#FF2A00:#FF5500:#FF8000:#FFAA00:#FFD500:#FFFF00:#D4FF00:#AAFF00:#80FF00:#55FF00:#2AFF00:#00FF00:#00FF2A:#00FF55:#00FF80:#00FFAA:#00FFD4:#00FFFF:#00D4FF:#00AAFF:#0080FF:#0055FF:#002BFF:#0000FF:#2A00FF:#5500FF:#8000FF:#AA00FF:#D500FF:#FF00FF:#FF00D4:#FF00AA:#FF0080:#FF0055:#FF002B'</default>
```

Comment:
Same here

Reply:
Take from my code whatever you like. My plan was only to make the **palette** working. This tool is in the _applications menu_ and should fully working!

I don't know anything about _wayland_ or which other tools use the _library_, but plan further rather heavy liftings of the **mate-colorsel*** stuff.

Reply:
I made a single commit, not changing the palette. resolves #643